### PR TITLE
{bio}[foss/2016b] AMOS v3.1.0

### PR DESCRIPTION
**Pull request by onkelmonokel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
closing and reopening to trigger travis

File: easybuild/easyconfigs/a/AMOS/AMOS-3.1.0-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+easyblock = 'ConfigureMake'
+
+name = 'AMOS'
+version = '3.1.0'
+
+homepage = 'http://sourceforge.net/apps/mediawiki/amos/index.php?title=AMOS'
+description = """The AMOS consortium is committed to the development of open-source whole genome assembly software"""
+
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/a/AMOS/AMOS-3.1.0-foss-2016b.eb]: style nitpicking: remove extra blank line

File: easybuild/easyconfigs/a/AMOS/AMOS-3.1.0-foss-2016b.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [('http://sourceforge.net/projects/amos/files/%s/%s' % (name.lower(), version), 'download')]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/a/AMOS/AMOS-3.1.0-foss-2016b.eb]: missing checksum

File: easybuild/easyconfigs/a/AMOS/AMOS-3.1.0_GCC-5.4.patch [hunk 3]
```diff
@@ -0,0 +1,11 @@
+diff -ru amos-3.1.0.orig/src/Align/find-tandem.cc amos-3.1.0/src/Align/find-tandem.cc
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/a/AMOS/AMOS-3.1.0_GCC-5.4.patch]: please add a description/motivation and author

**Comment by boegel** [comment]
Closing this, since `foss/2016b` is ancient history by now, and we have already have AMOS v3.1.0 with a more recent toolchain, see #7765...

Sorry for not following up on this better @onkelmonokel ... :(

**Pull request closed without merging**